lib/Runtime/Library/JSONStack.cpp

```diff
@@ -24,10 +33,18 @@ namespace JSON
     {
         if (bJsObject)
         {
+            if (STACK_VALUE)
+            {
+                return (data == STACK_VALUE);
```

Comment:
@kunalspathak  not sure I understood. there is an `if(STACK_VALUE)` why should I check STACK_VALUE != nullptr inside an Assert?